src/pattern-matching/let-control-flow.md

```diff
@@ -4,7 +4,7 @@ minutes: 10
 
 # Let Control Flow
 
-Rust has a few control flow constructs which differ from other languages. They
+Rust has a few control flow constructs, which differ from other languages. They
```

Comment:
```suggestion
Rust has a few control flow constructs that differ from other languages. They
```

"which" should be used to provide non-essential information. This paragraph, on the other hand, is trying to convey the idea that that Rust has some unique control flow constructs.